modules/pam_setquota/pam_setquota.8.xml

```diff
@@ -0,0 +1,239 @@
+          </term>
+          <listitem>
+            <para>
+              Hard limit for disk quota blocks, as defined by <citerefentry><refentrytitle>quotactl</refentrytitle><manvolnum>2</manvolnum></citerefentry>.
+            </para>
+          </listitem>
+	</varlistentry>
+	      <varlistentry>
+          <term>
+            <option>isoftlimit=<replaceable>3000</replaceable></option>
+          </term>
+          <listitem>
+            <para>
+              Soft limit for inodes, as defined by <citerefentry><refentrytitle>quotactl</refentrytitle><manvolnum>2</manvolnum></citerefentry>.
+            </para>
+          </listitem>
+	</varlistentry>
+	      <varlistentry>
+          <term>
+            <option>ihardlimit=<replaceable>4000</replaceable></option>
+          </term>
+          <listitem>
+            <para>
+              Soft limit for inodes, as defined by <citerefentry><refentrytitle>quotactl</refentrytitle><manvolnum>2</manvolnum></citerefentry>.
+            </para>
+          </listitem>
+	</varlistentry>
+      </variablelist>
+
+    </para>
+  </refsect1>
+
+  <refsect1 id="pam_setquota-types">
+    <title>MODULE TYPES PROVIDED</title>
+    <para>
+       Only the <option>session</option> module type is provided.
+    </para>
+  </refsect1>
+
+  <refsect1 id='pam_setquota-return_values'>
+    <title>RETURN VALUES</title>
+    <para>
+      <variablelist>
+
+        <varlistentry>
+          <term>PAM_SUCCESS</term>
+          <listitem>
+            <para>
+              The quota was set successfully or the UID of the user was outside of the specifed range.
+            </para>
+          </listitem>
+        </varlistentry>
+
+        <varlistentry>
+          <term>PAM_CRED_INSUFFICIENT</term>
+          <listitem>
+            <para>
+              The user was not found.
+            </para>
+          </listitem>
+        </varlistentry>
+
+        <varlistentry>
+          <term>PAM_PERM_DENIED</term>
+          <listitem>
+            <para>
+              <filename>/etc/mtab</filename> could not be opened
+              </para>
+              <para>
+               The filesystem specified was not found.
+               </para>
+               <para>
+               The limits for the user could not be retrieved. See syslog for more information.
+            </para>
+          </listitem>
+        </varlistentry>
+
+      </variablelist>
+    </para>
+  </refsect1>
+
+  <refsect1 id='pam_setquota-examples'>
+    <title>EXAMPLES</title>
+    <para>
+    A single invocation of `pam_setquota` applies a specific policy to a UID range. Applying different policies to specific UID ranges is done by invoking pam_setquota more than once. The last matching entry defines the resulting quota.
+      <programlisting>
+	session    required     /lib/security/pam_setquota.so bsoftlimit=1000 bhardlimit=2000 isoftlimit=1000 ihardlimit=2000 startuid=1000 enduid=0 fs=/home
```

Comment:
The examples are wrong, never use an absolute path, especially not one which is wrong on most systems.

Reply:
Right. I will fix this to be more in line with the documentation of the other modules.